src/onboarding/steps.ts

```diff
@@ -215,11 +217,16 @@ export function _getStepInfo({ firstScreenInStep, navigator, dispatch, props }:
     skipProtectWallet,
   } = props
 
+  function wrapNavigate<T extends keyof StackParamList>(screen: T, props: StackParamList[T]) {
```

Comment:
you can use the same type as the type of navigate from NavigationService (maybe they can be shared?) That way you can avoid the extra `undefined` in the calls below

Reply:
Ah yeah, I had tried this but encountered a strange typing problem. I think I've gotten it to work now, however